packages/broker/src/broker.ts

```diff
@@ -19,19 +24,28 @@ export interface Broker {
     stop: () => Promise<unknown>
 }
 
+const applyPluginClientConfigs = (plugins: Plugin<any>[], clientConfig: StrictConfig['client']) => {
```

Comment:
```suggestion
const applyPluginClientConfigs = (plugins: Plugin<unknown>[], clientConfig: StrictConfig['client']) => {
```

Reply:
Could use unit tests this function perhaps (since there is a bit of `any`  and `isEqual`) (Would be maybe good to test `isEqual` special cases: array / object).

Reply:
`unknown` type can't be used as the generics in Plugin is defined as `T extends `object

Reply:
Added unit tests